conn.go

```diff
@@ -196,35 +206,41 @@ func (c *conn) serveWrite() {
 	}
 }
 
+//todo maybe refactor this
```

Comment:
Either fix todo, remove todo, or add longer explanation what you're thinking here so future maintainers can work on it :)

Reply:
i will fix to it this release